grpc.go

```diff
@@ -935,9 +973,9 @@ func (a *agentGRPC) UpdateRoutes(ctx context.Context, req *pb.UpdateRoutesReques
 
 func (a *agentGRPC) OnlineCPUMem(ctx context.Context, req *pb.OnlineCPUMemRequest) (*gpb.Empty, error) {
 	if !req.Wait {
-		go onlineCPUMem(req)
+		go a.onlineCPUMem(req)
```

Comment:
This isn't strictly related to this PR (since the code already uses a go routine), but if `Wait` is `true`, fwics the agent will never log any errors from `onlineCPUMem()`.

But since `req` is passed to that function, it could log all error scenarios so atleast something is captured to the journal. It would make the code a bit less clean, but something like:

```
// handleError will log the specified error if wait is true
func handleError(wait bool, err error) error {
    if wait {
        agentLog.WithError(err).Error()
    }

    return err
}

func (a *agentGRPC) OnlineCPUMem(ctx context.Context, req *pb.OnlineCPUMemRequest) (*gpb.Empty, error) {

    :

    err := doSomething()
    if err != nil {
        return handleError(req.Wait, err)
    }

    :
}
```